### ISSUE TITLE
MeetingBar Segfault's shortly after starting up

### ISSUE THREAD

**Issue by jakeswenson** [leits/MeetingBar#236]
My MeetingBar is crashing today and I'm not sure why.
I started the app from the terminal and i can see it is getting a segmentation fault.

```bash
❯ /Applications/MeetingBar.app/Contents/MacOS/MeetingBar
2021-04-09 09:55:57.660 MeetingBar[54941:935726] Could not get real path for Address Book lock folder: open() for F_GETPATH failed.
2021-04-09 09:55:58.025 MeetingBar[54941:935724] Category EVENT was registered
2021-04-09 09:55:58.194 MeetingBar[54941:935724] Access to notications granted
2021-04-09 09:55:58.194 MeetingBar[54941:935724] Authorisation status 2
2021-04-09 09:55:58.194 MeetingBar[54941:935724] Alert Style settings 1
2021-04-09 09:55:58.195 MeetingBar[54941:935706] request <UNNotificationRequest: 0x7fc78e49c290; identifier: NEXT_EVENT, content: <UNNotificationContent: 0x7fc78e4a3bb0; title: <redacted>, subtitle: (null), body: <redacted>, summaryArgument: , summaryArgumentCount: 0, categoryIdentifier: EVENT, launchImageName: , threadIdentifier: meetingbar, attachments: (
), badge: (null), sound: <UNNotificationSound: 0x7fc78e42cc10>, realert: 0, trigger: <UNTimeIntervalNotificationTrigger: 0x7fc78e497dc0; repeats: NO, timeInterval: 236.805889>> was added
2021-04-09 09:55:58.431 MeetingBar[54941:935701] Loaded events for date 2021-04-09 from calendars ["Calendar"]
2021-04-09 09:55:58.437 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.438 MeetingBar[54941:935701] Returning text URL_SNIP
2021-04-09 09:55:58.506 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.506 MeetingBar[54941:935701] Returning text URL_SNIP
2021-04-09 09:55:58.509 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.509 MeetingBar[54941:935701] Returning text URL_SNIP
2021-04-09 09:55:58.511 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.512 MeetingBar[54941:935701] Returning text URL_SNIP
2021-04-09 09:55:58.515 MeetingBar[54941:935701] Check text Conference Room...; URL_SNIP for outlook safe links
2021-04-09 09:55:58.515 MeetingBar[54941:935701] Returning text Conference Room ...; URL_SNIP
2021-04-09 09:55:58.518 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.518 MeetingBar[54941:935701] Returning text URL_SNIP
2021-04-09 09:55:58.520 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.520 MeetingBar[54941:935701] Returning text URL_SNIP
2021-04-09 09:55:58.530 MeetingBar[54941:935701] Check text URL_SNIP for outlook safe links
2021-04-09 09:55:58.531 MeetingBar[54941:935701] Returning text URL_SNIP
[1]    54941 segmentation fault  /Applications/MeetingBar.app/Contents/MacOS/MeetingBar
```

Is there a way to enable more debugging/verbose output?

**Comment by leits** [comment]
Thanks for the report, @jakeswenson
Does this happen when you launch the app or when you try to join an event?

**Comment by jakeswenson** [comment]
When launching the app, it crashes right away.

**Comment by desmondfuller** [comment]
I'm having the same issue at Launch.  

`MeetingBar(2541,0x108aa7e00) malloc: Incorrect checksum for freed object 0x7f819a126e00: probably modified after being freed.
Corrupt value: 0x40022
MeetingBar(2541,0x108aa7e00) malloc: *** set a breakpoint in malloc_error_break to debug`

**Comment by leits** [comment]
I think, It was fixed in 3.2.2 which happened after this report, but it looks like I just forgot to close it.
@jakeswenson, @desmondfuller,  please, check the latest version!
